src/components/partnership/partnership.service.ts

```diff
@@ -81,10 +76,10 @@ export class PartnershipService {
       input.financialReportingType,
       input.types ?? [],
       partner,
-    );
+    ); // is this a check to be moved to the repository?
```

Comment:
I think from the comment below on `verifyFinancialReportingType` it was decided that this is okay here for now, but will be moved to the repo after migration.  So this comment can probably either be deleted or changed to a TODO similar to the one below.